client/disputes/new-evidence/index.tsx

```diff
@@ -397,6 +397,21 @@ export default ( { query }: { query: { id: string } } ) => {
 		setIsAccordionOpen( currentStep === 0 );
 	}, [ currentStep ] );
 
+	// Clear shipping information when shipping is not needed
+	useEffect( () => {
+		if ( ! hasShipping ) {
+			setShippingCarrier( '' );
+			setShippingDate( '' );
+			setShippingTrackingNumber( '' );
+			setShippingAddress( '' );
+			// Clear shipping documentation from evidence
+			setEvidence( ( prev: any ) => ( {
```

Comment:
Nothing major, but we could define a `BaseEvidence` type and use it here instead of `any`. This type could look something like:

```typescript
interface BaseEvidence {
  product_description: string;
  receipt: string;
  refund_policy: string;
  duplicate_charge_documentation: string;
  shipping_documentation: string;
  service_documentation: string;
  cancellation_policy: string;
  access_activity_log: string;
  uncategorized_file: string;
  uncategorized_text: string;
  shipping_carrier: string;
  shipping_date: string;
  shipping_tracking_number: string;
  shipping_address: string;
  customer_purchase_ip: string;
  // Add any additional fields as needed
}
``` 

Having types like this makes it a bit more convenient in the IDE.

Reply:
Hi @mgascam I addressed this on this commit f7bcfc5957b87cda5218edefc44cf43d59903699